{"name": "test_2_CreateLog", "status": "broken", "statusDetails": {"message": "TypeError: 'tuple' object is not callable", "trace": "self = <testcases.test_create_log.TestCreate_log object at 0x0443C6B8>\ntest_logsystem = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"30db3df7-feba-4457-aee1-80ec5e53a127\")>\n\n    def test_2_CreateLog(self,test_logsystem):\n        self.page1 = TestLogPage(test_logsystem)\n        self.page1.type_blog_title(\"我最帅\")\n        self.page1.type_tag(\"个人属性\")\n        self.page1.type_ke_content(\"我最帅，在座的各位谁赞成，谁反对？\")\n        time.sleep(2)\n>       self.page1.select_album()\nE       TypeError: 'tuple' object is not callable\n\ntest_create_log.py:33: TypeError"}, "start": 1628836713773, "stop": 1628836715826, "uuid": "5e07e632-59e4-4b47-aa0d-4b335c3ee64b", "historyId": "4cc768d0db640219c705049bf0abf532", "testCaseId": "b1dec58d31e6c5197e5bdc5714d1b0e5", "fullName": "testcases.test_create_log.TestCreate_log#test_2_CreateLog", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_create_log"}, {"name": "subSuite", "value": "TestCreate_log"}, {"name": "host", "value": "DESKTOP-S2QKEKE"}, {"name": "thread", "value": "18032-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_create_log"}]}